spec/factories/suspended_users.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :suspended_user, class: "Users::Suspended" do
+    transient do
```

Comment:
This makes it easy to create a suspended user with a specific username, e.g. 

```ruby
create(:suspended_user, username: "Test")
```

instead of having to mess around with the username hashing.